token-lending/program/src/state/obligation.rs

```diff
@@ -176,6 +176,21 @@ impl Obligation {
         Ok((&self.borrows[liquidity_index], liquidity_index))
     }
 
+    /// Find liquidity by borrow reserve mut
+    pub fn find_liquidity_in_borrows_mut(
```

Comment:
so only difference with non-mut version is this returns a mutable ObligationLiquidity so it can be updated with `accrue_interest`?

Reply:
yup

Reply:
Is there any reason why we don't just make the original one return a mutable variable?

I suppose theres a small risk of us wanting to change logic here and then only fixing it one of the places or something